BenchmarkSamples/benchmark/build.gradle

```diff
@@ -11,7 +11,16 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
 
+    testBuildType "benchmark"
+    buildTypes {
+        benchmark {
```

Comment:
If we don't specify other build types, do they still exist?

Reply:
I think debug and release are always there, but I updated it to debug anyway just to make this sample more clear :+1: